apps/studio/src/features/editing-experience/hooks/useTextEditor/constants.ts

```diff
@@ -76,7 +76,17 @@ export const BASE_EXTENSIONS: Extensions = [
   }),
   Dropcursor,
   Gapcursor,
-  HardBreak,
+  HardBreak.extend({
+    // Prevent the default behavior of the hard break and split it into a new paragraph instead
+    // Why: Hard break introduces touch targets of under 24px between lines
+    // This is a WCAG violation if the link is a clickable link
+    addKeyboardShortcuts() {
+      return {
+        "Mod-Enter": () => this.editor.commands.splitBlock(),
+        "Shift-Enter": () => this.editor.commands.splitBlock(),
+      }
```

Comment:
[question] i think our parsing rules for `ajv` for `prose` might destructively interfere with this rule (i can't quite remember what but i recall we had a rule for table that it can only have 1 header row or something similar)

could i trouble you to check the below sequence of steps
1. enable header col/row for table
2. use mod or shift enter to split the table into 2 
3. for the new table below, **also enable header col/row**
4. now, delete the hard break -> does this cause validation issues?

Reply:
re: convo

no longer relevant as users can no longer add new header col/row on prose blocks